shell/platform/android/io/flutter/view/AccessibilityBridge.java

```diff
@@ -422,6 +415,31 @@ void updateSemantics(ByteBuffer buffer, String[] strings) {
             }
         }
 
+        // Now that the tree is pruned, sort all the entries and set their
+        // previous ids.
+        it = mObjects.entrySet().iterator();
```

Comment:
So, on any semantics update we have to re-sort the entire tree? I am a little concerned about performance...